style.css

```diff
@@ -0,0 +1,67 @@
+#main {
+    margin: 30px 8px;
+    padding: 15px;
+    border-radius: 5px;
+    background: #eee;
```

Comment:
please use linters for correction you can run  "npx stylelint "**/*.{css,scss}" to see what changes you can make. 😉